Source/Core/VideoCommon/BPMemory.h

```diff
@@ -765,7 +765,7 @@ enum class MaxAnsio
 template <>
 struct fmt::formatter<MaxAnsio> : EnumFormatter<MaxAnsio::Four>
```

Comment:
something that you might want to fix in a follow-up PR: MaxAn**is**o seems to be misspelled

Reply:
Thanks; I'll add a fix for that to #9718.